File: TechRadar.Services/Models/Quadrant.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace TechRadar.Services.Models
{
    [BsonIgnoreExtraElements]
    public class Quadrant
    {
        [BsonId]
        public ObjectId Id { get; set; }

        [BsonElement("quadrantNumber")]
        public int QuadrantNumber { get; set; }

        [BsonElement("name")]
        public string Name { get; set; }

        [BsonElement("description")]
        public string Description { get; set; }
    }
}

File: TechRadar.Web/ClientApp/app/models/quadrant.ts
import { Blip } from './blip';

export class Quadrant {
    id: string;
    quadrantNumber: number;
    name: string;
    description: string;

    constructor(quandrantInfo: any) {
        this.id = quandrantInfo.id;
        this.quadrantNumber = quandrantInfo.quadrantNumber;
        this.name = quandrantInfo.name;
        this.description = quandrantInfo.description;
    }
}
